metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -298,6 +308,56 @@ def make_tableau_client(self, site: str) -> Server:
             ) from e
 
 
+class AccessRolesIngestionConfig(ConfigModel):
+    enable_workbooks: bool = Field(
+        default=True,
+        description="Whether or not to enable access role ingestion for workbooks. "
+        "Default: True",
+    )
+
+    group_name_pattern: AllowDenyPattern = Field(
+        default=AllowDenyPattern.allow_all(),
+        description="Filter for Tableau group names when ingesting access roles. "
+        "For example, you could filter for groups that include the term 'Consumer' in their name by adding '^.*Consumer$' to the allow list."
+        "By default, all groups will be added as roles. "
+        "You can both allow and deny groups based on their name using their name, or a Regex pattern. "
+        "Deny patterns always take precedence over allow patterns. ",
+    )
+
+    role_prefix: str = Field(
+        default="",
+        description="Prefix to append to the group name when generating the role name used in the request_url.",
+    )
+
+    group_substring_start: Optional[int] = Field(
+        default=None,
+        description="Use this property if you only need a substring of the group in Tableau to create the role used in the request_url.",
+    )
+
+    group_substring_end: Optional[int] = Field(
+        default=None,
+        description="Use this property if you only need a substring of the group in Tableau to create the role used in the request_url.",
+    )
+
+    role_description: str = Field(
+        default="",
+        description="Description of the role that will be displayed in the Access Management tab of Datahub.",
+    )
+
+    displayed_capabilities: Optional[List[str]] = Field(
```

Comment:
I'm considering a suitable name for this configuration. How do you feel about `displayed_permissions` or `visible_permissions`?

Reply:
In Tableau this is called a Capability:
![image](https://github.com/user-attachments/assets/43b4762f-387e-490f-813e-a5aec5ebff4d)
See: https://tableau.github.io/server-client-python/docs/api-ref or https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_ref_permissions.htm#delete_workbook_permission. I'd keep it as is but let me know what you think or if you want me to change it to something else.